src/pages/index.js

```diff
@@ -18,28 +20,77 @@ const profileTitleInput = document.querySelector("#profile-title-input");
 const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
+const profileImageContainer = document.querySelector(
+  ".profile__image-container"
+);
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "fd75a5c6-0c34-4591-90ec-6f1fc1e0d0b3",
+    "Content-Type": "application/json",
+  },
+});
 
+profileImageContainer.addEventListener("click", () => {
+  editAvatarPopup.open();
+});
 const addCardFormValidator = new FormValidator(configItems, addCardFormElement);
 const profileEditValidator = new FormValidator(configItems, profileEditForm);
+const editAvatarFormValidator = new FormValidator(
+  configItems,
+  document.querySelector("#avatar-edit-form")
```

Comment:
`document.querySelector("#avatar-edit-form")` is searched 2 times. Please, define `avatarForm` only once

It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine